rest_framework_swagger/introspectors.py

```diff
@@ -597,20 +597,34 @@ def build_query_parameters(self):
         parameters = super(ViewSetMethodIntrospector, self) \
             .build_query_parameters()
         view = self.create_view()
-        if not hasattr(view, 'paginate_by'):
-            return parameters
+        if rest_framework.VERSION >= '3.0.1':
```

Comment:
Perhaps factor this out into some compat module rather than putting the logic right here.